sci-libs/opencascade/opencascade-7.8.0.recipe

```diff
@@ -335,6 +347,7 @@ BUILD()
 		-D3RDPARTY_QT_DIR=$portPackageLinksDir/devel~libqt5 \
 		-D3RDPARTY_SKIP_DOT_EXECUTABLE=OFF \
 		-DBUILD_Inspector=ON \
+		-DBUILD_OPT_PROFILE=Production \
```

Comment:
This is a new build option for enabling LTO (link time optimization). Is this a good idea to use?